_dashboards/dashboard/plugins-dashboards.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: default
+title: Integrating plugins into a dashboard
+parent: Observability
+nav_order: 5
+---
+
+# Integrating plugins into a dashboard
+
+Observability is a collection of plugins and applications that let you visualize data-driven events by using [Piped Processing Language]({{site.url}}{{site.baseurl}}/search-plugins/sql/ppl/index/) to explore, discover, and query data stored in OpenSearch. Observability provides a unified experience for collecting and monitoring metrics, logs, and traces from common data sources. With data collection and monitoring in one place, you have full-stack, end-to-end observability of your entire infrastructure. 
+
+As of OpenSearch 2.7, you can manage your observability plugins with **Observability Dashboards** or **Dashboard**, instead of the plugins page. This feature provides you:
```

Comment:
```suggestion
As of OpenSearch 2.7, you can manage your observability plugins with **Observability Dashboards** or **Dashboard** instead of the plugins page. This feature provides you:
```